automation/README.md

```diff
@@ -0,0 +1,8 @@
+
+# Run all UI tests
+- cd automation/
+- npm test
```

Comment:
Can we run through yarn? Just to stay consistent.

Reply:
File is not updated. yarn test works and it’s what I use to run test.

Reply:
👍 sounds good.